addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1209,6 +1209,13 @@ registry.WebsiteAnimate = publicWidget.Widget.extend({
         // By default, elements are hidden by the css of o_animate.
         // Render elements and trigger the animation then pause it in state 0.
         this.$animatedElements = this.$target.find('.o_animate');
+        // Fix for "transform: none" not overriding keyframe transforms on
+        // iPhone 8 and lower.
+        this.o_force_overflow_x_hidden = false;
+        if (this.$animatedElements[0] && window.getComputedStyle(this.$animatedElements[0]).transform !== 'none') {
+            this._toggleOverflowXHidden(true);
+            this.o_force_overflow_x_hidden = true;
+        }
```

Comment:
Ok, let's go with that, although: why starting the name of your variable by `o_` and using snake case? :grimacing:

Reply:
> Ok, let's go with that, although: why starting the name of your variable by `o_` and using snake case? grimacing

I think it's because my brain was in "css" mode. I changed that :+1: